packages/@react-spectrum/s2/src/ContextualHelp.tsx

```diff
@@ -40,9 +40,8 @@ export interface ContextualHelpProps extends
 }
 
 const popover = style({
-  fontFamily: 'sans',
-  minWidth: 268,
-  width: 268,
+  minWidth: 'calc((268 / 16 * var(--s2-scale) - (2 * 1 / 16)) * 1rem)',
+  width: 'calc((268 / 16 * var(--s2-scale) - (2 * 1 / 16)) * 1rem)',
```

Comment:
tried using the size util in combination with the border css var but for some reason using that would bork the calc... Manually calculating for now